packages/hooks/src/useEventListener.ts

```diff
@@ -40,14 +40,34 @@ export default function useEventListener<Type extends keyof DocumentEventMap>(
       return;
```

Comment:
in the line above, I'm wondering if we should be more explicit and say `enabled === false` to early return. Seems like we get that behavior due to fortune that `'once'` is truthy. 

Don't feel strongly about this tho.